.gitlab-ci/lint.yml

```diff
@@ -64,9 +64,10 @@ markdownlint:
   tags: [light]
   image: node
   before_script:
-    - npm install -g markdownlint-cli
+    - npm install -g markdownlint-cli@0.22.0
```

Comment:
The latest stable markdownlint is 0.22.0 as https://github.com/DavidAnson/markdownlint/tags
However if not specifying any version on markdownlint, 0.26.0 is installed and it outputs the following error locally:
```
$ markdownlint --version
/usr/local/lib/node_modules/markdownlint-cli/markdownlint.js:52
    } catch {
            ^

SyntaxError: Unexpected token {
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:616:28)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:188:16)
    at bootstrap_node.js:609:3
oomichi@dev:~/go/src/k8s.io/kubespray$ markdownlint version
/usr/local/lib/node_modules/markdownlint-cli/markdownlint.js:52
    } catch {
            ^

SyntaxError: Unexpected token {
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:616:28)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:188:16)
    at bootstrap_node.js:609:3
```
So this specifies the latest stable version.